### PR TITLE
Merger

### PR DESCRIPTION
**Pull request by Dolezal-557280**
This short function should combine the results of the two models, numerical and textual, and provide an output in %. The function can later be adjusted so that the best possible output is provided (average, min/max).

### CODE REVIEW

File: merger [hunk 1]
```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
+    # 1. Combine the two numbers into an average value
+    average_value = (textual_chance + numerical_chance) / 2
+
+    # 2. Display the median of the two values
+    median_value = sorted([textual_chance, numerical_chance])[1]  # Since there are only two values, the second one is the median
```
Comment (janrygl) [inline merger]: odd count ... middle value
even count .. average of two mid values
...
for two values, median = average

File: merger [hunk 2]
```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
+    # 1. Combine the two numbers into an average value
+    average_value = (textual_chance + numerical_chance) / 2
+
+    # 2. Display the median of the two values
```
Comment (janrygl) [inline merger]: Output instead of display, display is for printing/GUIs, outputting/returning is for functions

File: merger [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    median_value = sorted([textual_chance, numerical_chance])[1]  # Since there are only two values, the second one is the median
+
+    # 3. Show the maximum value
```
Comment (janrygl) [inline merger]: same as display, also comment is not needed, `max_value` is readable as is

File: merger [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    max_value = max(textual_chance, numerical_chance)
+
+    # 4. Show the minimum value
+    min_value = min(textual_chance, numerical_chance)
+
+    # Display results
+    print(f"Average Value: {average_value:.2f}%")
+    print(f"Median Value: {median_value:.2f}%")
+    print(f"Maximum Value: {max_value:.2f}%")
+    print(f"Minimum Value: {min_value:.2f}%")
```
Comment (janrygl) [inline merger]: I suggest to output dict of values, e.g.
```
return {
    "Average Value": average_value,
...
}
```

and then make another function `print_results(data: dict)`
which make this nice print

Final call could be
```
print_results(combine_chances(textual_chance, numerical_chance))
```

File: merger [hunk 5]
```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
```
Comment (janrygl) [inline merger]: lets allow any number of arguments, not just 2

simple syntax approach
```
def combine_chances(changes: list[float]):

combine_chances([0.2, 0.1, 0.445])
```

or advanced syntax approach:

```
def combine_chances(*args):

combine_chances(0.2, 0.1, 0.445)
```

**Review by janrygl — CHANGES_REQUESTED** [review]
Nice basic printer function.
Useful, but please:
- fix wrong median calculation
- separate into two functions: one calculating and one printing

Well done

File: merger (hunk 3 above)
Comment (PavelOujesky) [reply merger]: the same for min_value

**Pull request merged**